plugins/org.locationtech.udig.catalog.tests/.classpath

```diff
@@ -2,6 +2,10 @@
 <classpath>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
-	<classpathentry kind="src" path="src/"/>
+	<classpathentry kind="src" path="src/">
+		<attributes>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>

```

Comment:
What option lead to the classpath entry, just "Plug-In Tools -> Update classpath"?

Reply:
Yes, exactly :) Should I revert this change as it does not belong to the PR anymore.

Reply:
No, please at it as it is and IMHO it belongs to this changeset.